app/controllers/registrations_controller.rb

```diff
@@ -72,13 +72,14 @@ def after_sign_up_path_for(_resource)
     def check_registration_limit
       timeframe = (ENV["MAMPF_REGISTRATION_TIMEFRAME"] || 15).to_i.minutes
       threshold_date = DateTime.now - timeframe
-      new_users = User.where(
+      num_new_registrations = User.where(
         "users.confirmed_at is NULL and users.created_at > '#{threshold_date}'"
       ).count
```

Comment:
If we are very strict: This query violates sveral cops, but this is probably not recognized by RuboCop due to the complexity:
https://github.com/rubocop/rails-style-guide#avoid-interpolation
https://github.com/rubocop/rails-style-guide#hash-conditions
https://github.com/rubocop/rails-style-guide#where-ranges

Reply:
Perhaps something like
```
timeframe = (ENV["MAMPF_REGISTRATION_TIMEFRAME"] || 15).to_i.minutes.ago
num_new_rgistration = User.where(confirmed_at: nil, created_at: timeframe..).count
````
or similar would be better?